extensions/strategies/dema-lessnoise/strategy.js

```diff
@@ -0,0 +1,83 @@
+var z = require('zero-fill')
+  , n = require('numbro')
+
+module.exports = function container (get, set, clear) {
+  return {
+    name: 'dema',
+    description: 'Buy when (short ema > long ema) and sell when (short ema < long ema).',
```

Comment:
How is it "lessnoise"? Also I think you need to update the name here?

Reply:
Just FYI, using VMA as long MA will filter noise better, because of it adjusts volatility. Try EMA10/VMA10, on 15 or 30 min intervals. Of cause it works only in strong trends.